subsys/sip_svc/sip_svc_intel_socfpga.c

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
I don't have a strong opinion on this, but should this file (containing the platform-specific functions) belong to the `driver/svc/`? The `subsys/svc` is designed to contain the most common, no-platform-specific, and abstracted interface, right? What's your opinion?

Reply:
I have made the change to bring the platform specific implementation to the driver side , can you please check if this is ok?

Reply:
Yes, LGTM